### PR TITLE
PLAT-97905: Add sampler to moonstone

### PR DESCRIPTION
**Pull request by germboy**
Adds the `samples/sampler` folder structure to `moonstone`. This adds the existing samples and qa-samples.

### CODE REVIEW

**Comment by germboy** [comment]
This change is dependent on the `enactjs/storybook-utils` repo added in: https://github.com/enactjs/storybook-utils/pull/1

**Comment by ryanjduffy** [comment]
@germboy - We should add a configuration option to ignore test coverage for the various sample and test folders. I think the following is what we need but please work with @JayCanuck to confirm.

```js
// CLI jest.config.js update
const ignorePatterns = [
	// Common directories to ignore
	'/node_modules/',
	'<rootDir>/(.*/)*coverage/',
	'<rootDir>/(.*/)*build/',
	'<rootDir>/(.*/)*dist/',
	'<rootDir>/(.*/)*samples/', // sampler, et al
	'<rootDir>/(.*/)*tests/screenshot/', // screenshot tests
	'<rootDir>/(.*/)*tests/ui/' // ui tests
];
```

Also, can you summarize the changes made in the migration in the description above? Everything works okay from my spot checking but the volume of code makes it difficult to review in detail without knowing where to look.

**Review by webOS101 — APPROVED** [review]
